arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/CreateSection.java

```diff
@@ -195,30 +195,33 @@ public CreateSection(Hub hub) {
           new ShakiraDataOperation(hub.stamp(), create2InitCode);
       hub.shakiraData().call(shakiraDataOperation);
 
-      triggerHashInfo(shakiraDataOperation.result());
+      writeHashInfoResult(shakiraDataOperation.result());
```

Comment:
Here the distinction is: we may want to _write_ a hash result without triggering HASH_INFO. This is for instance the case when doing
- a KECCAK with zero size or
- an (unexceptional, unaborted) CREATE2 with zero init code size

I change the method to **not** set `triggerHashInfo` anymore but to simply write the HASH to the relevant cells. Thus the name change.